sycl/test-e2e/Matrix/joint_matrix_apply_two_matrices.cpp

```diff
@@ -10,5 +10,7 @@
 // RUN: %{build} -ffp-model=precise -o %t.out
 // RUN: %{run} %t.out
 
+// XFAIL: gpu-intel-dg2
+
```

Comment:
same as above:
Adding XFAIL will make the code fail silently for the other supported combination.